app/src/main/res/values-in/strings.xml

```diff
@@ -205,23 +211,28 @@
     <string name="reboot">Nyalakan ulang</string>
     <string name="release_notes">Catatan rilis</string>
     <string name="repo_cache_cleared">Cache repo dihapus</string>
-    <string name="flashing">Memasang…</string>
+    <string name="flashing">Flashing…</string>
     <string name="done">Selesai!</string>
     <string name="failure">Gagal!</string>
+    <string name="hide_app_title">Menyembunyikan aplikasi Magisk…</string>
     <string name="open_link_failed_toast">Tidak ditemukan aplikasi untuk membuka link ini</string>
     <string name="complete_uninstall">Uninstal penuh</string>
     <string name="restore_img">Pulihkan image</string>
     <string name="restore_img_msg">Memulihkan…</string>
     <string name="restore_done">Pemulihan selesai!</string>
     <string name="restore_fail">Cadangan stock tidak ada!</string>
-    <string name="proprietary_title">Download kode proprieter</string>
+    <string name="proprietary_title">Download kode proprietor</string>
+    <string name="proprietary_notice">Magisk adalah software FOSS dan tidak menyertakan kode proprietor API SafetyNet Google.\n\nApakah Anda ingin mendownload ekstensi proprietor untuk melakukan pemeriksaan SafetyNet?</string>
```

Comment:
FOSS implies *software*, unless you want to translate the whole FOSS to "perangkat lunak bebas dan bersumber terbuka"